examples/axes_grid1/scatter_hist_locatable_axes.py

```diff
@@ -21,14 +31,16 @@
 
 # the scatter plot:
 axScatter.scatter(x, y)
+
+# Set aspect of the main axes.
 axScatter.set_aspect(1.)
 
 # create new axes on the right and on the top of the current axes
-# The first argument of the new_vertical(new_horizontal) method is
-# the height (width) of the axes to be created in inches.
 divider = make_axes_locatable(axScatter)
-axHistx = divider.append_axes("top", 1.2, pad=0.1, sharex=axScatter)
-axHisty = divider.append_axes("right", 1.2, pad=0.1, sharey=axScatter)
+axHistx = divider.append_axes("top", 1.2, pad=0.1,  # height and pad in inches
```

Comment:
Would the plots look better with `pad=0`?

Reply:
I don't think they look bad in the first place. With pad=0 the axis of the main plot would directly continue into the independent axis of the marginal plot; hence one would need to introduce another visual distinction.